src/Sylius/Bundle/ApiBundle/Resources/config/filters.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ This file is part of the Sylius package.
+ (c) Paweł Jędrzejewski
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="Sylius\Bundle\ApiBundle\Filters\ExchangeRateFilter">
+            <tag name="api_platform.filter" id="exchange_rate_filter"/>
```

Comment:
Have you check convention in ApiPlatform itself?

Reply:
Yes, its not working without tag name and tag id, Configuration in ApiPlatform docs is available only for yaml and annotation, I not saw tag id there, but I investigate that its not working without